ui/redux/reducers/settings.js

```diff
@@ -2,12 +2,18 @@ import * as ACTIONS from 'constants/action_types';
 import * as SETTINGS from 'constants/settings';
 import moment from 'moment';
 import SUPPORTED_LANGUAGES from 'constants/supported_languages';
+import { ACTIONS as LBRY_REDUX_ACTIONS, SHARED_PREFERENCES } from 'lbry-redux';
 
 const reducers = {};
 const defaultState = {
   isNight: false,
   loadedLanguages: [...Object.keys(window.i18n_messages), 'en'] || ['en'],
+  customWalletServers: [],
+  sharedPreferences: {
+    [SHARED_PREFERENCES.WALLET_SERVERS]: false,
```

Comment:
Should this be something other than false? Maybe an empty array or object? At least undefined.

What does this value look like when it's populated?

Reply:
right, js object {} dictionary.